DICOMPlugins/base/DICOMM3DPluginBase.py

```diff
@@ -0,0 +1,92 @@
+import slicer
+import pydicom
+import logging
+from datetime import datetime
+from DICOMLib import DICOMPlugin
+import shutil
+
+
+class DICOMM3DPluginBase(DICOMPlugin):
```

Comment:
I understood this code is already in QuantitativeReporting so can you re-use that?

Reply:
yes this .py script is a duplicate of QuantitativeReporting/DICOMPluginBase.py, with a few added attributes, specific to M3D DICOM, i did not what was correct here, have everything in the same DICOMPlugin class or separate ones for M3D and SEG DICOM.

Reply:
My feeling is that there's not enough to justify another base class here.  There are not likely to be any other classes that derive from this `DICOMM3DPluginBase` so to me it makes more sense to put anything specific to M3D into the plugin class itself.

Reply:
I agree!